### DISCUSSION TITLE
Cards layout : blank target

### DISCUSSION THREAD

**Discussion by tocri** [microsoft-search/pnp-modern-search#3703] [Q&A] [answered]
Hi, i would like to open my cards in a new tabs. I've read that i need to add the html _**_target="_blank" data-interception="off"_**_
But my title hndlebar doesn't allow html custom

![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/7407992b-f203-4f28-a3dc-b795f455e496)

Could you clarify please. I'm still a newbie

**Comment by wobba** [comment]
We are using the fluentui react `DocumentTitle` component which does not support custom HTML, only pure text.

**Comment by tocri** [comment]
> We are using the fluentui react `DocumentTitle` component which does not support custom HTML, only pure text.

thank you. I understand but how i change the default behavoir of a card layout ? Need i change the layout ?

**Reply by wobba** [reply, marked as answer]
The card layout support the slots available. If you want something outside of this you can use a custom layout.

**Reply by tocri** [reply]
Ok thanks to you and for the quick reply

**Reply by tocri** [reply]
Ok find it.

I lost about flexibility but the default behaviour is what i expected.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/b5f0dd1c-0d9a-4789-b417-6ce57b13a38e)

<img width="464" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/10110260/7c14c3dc-c96c-494d-8ab7-695ec6ad9b59">
